src/bindings/python/tests/test_graph/test_adaptive_pool.py

```diff
@@ -1,69 +0,0 @@
-# -*- coding: utf-8 -*-
-# Copyright (C) 2018-2022 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-import openvino.runtime.opset8 as ov
-import numpy as np
-from tests.runtime import get_runtime
-
-
-def test_adaptive_avg_pool():
```

Comment:
Tests from this file are already covered in https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_graph/test_create_op.py, both Adaptive Max Pool and Adaptive Average Pool.